filterNotam.py

```diff
@@ -77,4 +77,28 @@ def identify_lighting_marking_notams(notams):
 
 # This function will filter out NOTAMs that have been marked as lighting or marking NOTAMs.
 def filter_out_lighting_marking_notams(notams, marked_notams):
-    return [notam for notam in notams if notam not in marked_notams]
\ No newline at end of file
+    return [notam for notam in notams if notam not in marked_notams]
+
+def filter_classification(notams, classification):
+    marked_notams = set()
+    for notam in notams:
+        if classification in notam.classification:
+            marked_notams.add(notam)
+    return marked_notams
```

Comment:
like the function above you could do this succinctly with Python list/set comprehensions:
```python
return { notam for notam in notams if classification in notam.classification }
```

Reply:
fixed in latest commit